### DISCUSSION TITLE
请问能否修改momos的日期

### DISCUSSION THREAD

**Discussion by oslytherin** [usememos/memos#166] [Q&A] [answered]
请问能否修改momos的日期，方便补录或者修改呢？

**Comment by boojack** [comment, marked as answer]
Updated with commit: https://github.com/usememos/memos/commit/57f51d1c58980de9f6c9fde6fa76a2a966eeb56a, and will be released in next version `v0.4.1`.

https://user-images.githubusercontent.com/24653555/186686527-225787a4-87db-44ea-ad21-bfc424a19bd3.mov

**Reply by oslytherin** [reply]
OK，thanks
